unsupportedCharacter.md

```diff
@@ -0,0 +1,44 @@
+# Unsupported Character detected in request
+
+Unsupported character detected in request sent to server. 
```

Comment:
Should explain what characters we currently accept in this so the customer can fix it